### PR TITLE
Detoxtests

### PR DESCRIPTION
**Pull request by fluidvanadium**
third time's the charm. this should pass tests

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -8,6 +8,7 @@ buildscript {
         compileSdkVersion = 31
         targetSdkVersion = 31
         ndkVersion = "25.1.8937393"
+        kotlinVersion = '1.7.10'
```
Comment (juanky201271) [inline android/build.gradle:10]: As you can see there is a duplicate with `kotlinversion`, you can safely remove the line number 4 in this file.

File: app.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
-  "name": "Zingo!",
+  "name": "ZingoMobile",
```
Comment (juanky201271) [inline app.json]: upsss, did you need that this `name` is `ZingoMobile`???? for some reason...

**Review by juanky201271 — CHANGES_REQUESTED** [review]
I have a couple of suggestions, overall seems perfect.

File: app.json (hunk 2 above)
Comment (fluidvanadium) [reply app.json]: Screenshot from 2023-02-03 16-25-17

that fixed this bug in yarn react-native start

File: android/build.gradle (hunk 1 above)
Comment (fluidvanadium) [reply android/build.gradle:10]: fixed

File: e2e/README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+to run e2e tests:
+
+>$ yarn start react-native start
+
```
Comment (Oscar-Pepper) [inline e2e/README.md:6]: i cant run these e2e tests on my setup, i suspect there is a yarn detox build step maybe?
i will post when i have it up and running for more info

**Review by Oscar-Pepper — CHANGES_REQUESTED** [review]
see comment in README

File: app.json (hunk 2 above)
Comment (juanky201271) [reply app.json]: this is strange... this is my console... (i have no problems with `Zingo!`...)
image

File: e2e/README.md (hunk 3 above)
Comment (Oscar-Pepper) [reply e2e/README.md:6]: ok I got this running by:
1. yarn detox build --configuration android.att.debug
2. start the Pixel_6_API_30 in the emulator
3. yarn detox test --configuration android.att.debug

Comment (Oscar-Pepper) [reply e2e/README.md:6]: yarn detox test --configuration android.emu.debug
seems to run step 2 and 3 in one step ----^

Comment (Oscar-Pepper) [reply e2e/README.md:6]: please atleast add this line before line 6 (yarn test):
yarn detox build --configuration android.att.debug

File: android/app/src/main/res/xml/network_security_config.xml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <domain-config cleartextTrafficPermitted="true">
+        <domain includeSubdomains="true">10.0.2.2</domain>
```
Comment (juanky201271) [inline android/app/src/main/res/xml/network_security_config.xml:4]: @fluidvanadium What means this line? I'm curious... It's a white list of IPs???

**Comment by AloeareV** [comment]
Superseded by #180

**Pull request closed without merging**